src/dmd/root/outbuffer.d

```diff
@@ -28,7 +28,12 @@ struct OutBuffer
     private ubyte[] data;
     private size_t offset;
     private bool notlinehead;
+
+    /// Whether to indent
     bool doindent;
+    /// Whether to indent by 4 spaces or by tabs;
+    bool spaces;
+    /// Current indent levlel
```

Comment:
level

Reply:
why this is extremely unlikely to ever break.
And it's going to be tested in dtoh the tests.
Therefore a unit-test is just a waste to everyone's time if they have to wait for the auto-tester to finish.

Reply:
> Therefore a unit-test is just a waste to everyone's time if they have to wait for the auto-tester to finish.

A unittest is rarely a waste of everyone's time. `auto-merge` is here for a reason. Please don't systematically disregard the review comments you are getting.